packages/reason-relay/language-plugin/src/__tests__/__snapshots__/languagePlugin-tests.ts.snap

```diff
@@ -268,9 +256,9 @@ module Types = {
 module Internal = {
   type wrapResponseRaw;
   let wrapResponseConverter: Js.Dict.t(Js.Dict.t(Js.Dict.t(string))) = [%raw
-    {json| {\\"__root\\":{\\"setUserLocation_changedUser\\":{\\"n\\":\\"\\"},\\"setUserLocation_changedUser_role\\":{\\"e\\":\\"enum_UserRole\\"}}} |json}
+    {json| {\\"__root\\":{\\"setUserLocation_changedUser\\":{\\"n\\":\\"\\"}}} |json}
   ];
-  let wrapResponseConverterMap = {\\"enum_UserRole\\": wrap_enum_UserRole};
+  let wrapResponseConverterMap = ();
```

Comment:
It's saving an object allocation, but that doesn't really matter too much I guess. It could be anything as nothing on it will be called when there are no instructions, which is why I made it unit. But maybe it's clearer if it's just an empty object.

Reply:
I see, I was just wondering how this could type check, as long as it's documented somewhere I think it's ok, maybe `None` would be a bit clearer then? Or maybe a few empty object allocations are not such a big deal, I have no idea, you know better :)